package.json

```diff
@@ -24,17 +24,11 @@
     "@cyclonedx/cyclonedx-npm": "1.12.1",
     "@types/jest": "29.5.2",
     "@types/node": "20.3.3",
-    "@typescript-eslint/eslint-plugin": "5.60.1",
```

Comment:
Moving all eslint dependencies into `eslint-config`

Reply:
I left this out because the [ESLint docs](https://eslint.org/docs/latest/extend/shareable-configs) seemed to suggest that it should be a `peerDependency`:
> If your shareable config depends on a plugin, you should also specify it as a peerDependency (plugins will be loaded relative to the end user’s project, so the end user is required to install the plugins they need). However, if your shareable config depends on a [custom parser](https://eslint.org/docs/latest/extend/custom-parsers) or another shareable config, you can specify these packages as dependencies in the package.json.

Reply:
Ahh, good catch!